meta-mender-imx/recipes-bsp/u-boot-imx/u-boot-imx_%.bbappend

```diff
@@ -1,6 +1,10 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
+require recipes-bsp/u-boot/u-boot-mender.inc
```

Comment:
Can you use this syntax instead?

`include ${@mender_feature_is_enabled("mender-uboot","u-boot-mender.inc","",d)}`

That matches with what is in master/dunfell better

Reply:
@drewmoseley done